package.json

```diff
@@ -136,14 +136,14 @@
         "properties": {
           "vsflux.defaultInfluxDBURLs": {
             "default": [
+              "http://localhost:9999",
               "https://us-central1-1.gcp.cloud2.influxdata.com",
```

Comment:
What is `default` for? All of these should be v2 connections.

Reply:
I believe it means what the default for config will be if they are not set explicitly (which we don't have ui for) so this is just the list.

Reply:
this list only gets used for v2 connection also